tests/test_reader.py

```diff
@@ -575,18 +575,18 @@ def test_issue604(strict):
     """Test with invalid destinations"""  # todo
     with open(os.path.join(RESOURCE_ROOT, "issue-604.pdf"), "rb") as f:
         pdf = None
-        bookmarks = None
+        outline = None
         if strict:
             pdf = PdfReader(f, strict=strict)
             with pytest.raises(PdfReadError) as exc, pytest.warns(PdfReadWarning):
-                bookmarks = pdf._get_outlines()
+                outline = pdf._get_outline()
```

Comment:
Could we replace this with `pdf.outline` to avoid the internal call? Same with some of the internal calls in the tests.